torch/_inductor/ir.py

```diff
@@ -3670,6 +3675,21 @@ def is_not_write(arg):
             x.name for x in kernel._schema.arguments if x.kwarg_only
         ]
 
+    def get_args_value(self, pos):
```

Comment:
`get_arg_default_value` is more appropriate for what it's doing?

Reply:
Renamed the function.